server/src/main/java/ru/hse/server/exception/UserAlreadyExistException.java

```diff
@@ -0,0 +1,7 @@
+package ru.hse.server.exception;
+
+public class UserAlreadyExistException extends RuntimeException {
+    public UserAlreadyExistException(String message){
```

Comment:
formatting

just to set checkbox "checking formatting before commit" in your ide